### ISSUE TITLE
No longer maintained

### ISSUE THREAD

**Issue by EvanHahn** [EvanHahn/cyborg.txt#4]
I am no longer maintaining this library. If you would like to take over, please comment!

**Comment by selfagency** [comment]
Hey Evan,

I'll take it over if you'd like.

**Comment by EvanHahn** [comment]
@selfagency Sure! Would you mind sending me an email at me@evanhahn.com and we'll get things sorted?

**Comment by EvanHahn** [comment]
@selfagency Didn't get anything...did you send it to me@evanhahn.com?

**Comment by selfagency** [comment]
yep, but my smtp seems to be hit or miss these days

**Comment by EvanHahn** [comment]
@selfagency Still haven't received anything. Feel free to shoot me a [Twitter DM](https://twitter.com/EvanHahn) or a message [on Gitter](https://gitter.im/helmetjs/helmet) if that works better.

**Comment by EvanHahn** [comment]
@selfagency Are you still interested in taking over this project?

**Comment by selfagency** [comment]
Yes, however, I left GitHub ([see why here](https://github.com/selfagency/microsoft-drop-ice)) and would want to move the library to Bitbucket or another service. Would that be ok? I'd need you to transfer ownership to me here and also on NPM.

**Comment by EvanHahn** [comment]
Sure—I respect that. Would you mind shooting me an email at me@evanhahn.com and we can discuss all this stuff?
